.github/workflows/preDeploy.yml

```diff
@@ -30,10 +31,15 @@ jobs:
         with:
           GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
 
+      - name: Check if merged pull request was an automatic version bump PR
+        run: echo "::set-output name=IS_VERSION_BUMP_PR::${{ constains(steps.getMergedPullRequest.outputs.labels, automerge) && github.actor == 'OSBotify' }}"
+
   skipDeploy:
     runs-on: ubuntu-latest
     needs: chooseDeployActions
-    if: ${{ needs.chooseDeployActions.outputs.isStagingDeployLocked == 'true' }}
+    if: |
```

Comment:
Tested this syntax for multiline if statements [here](https://github.com/Andrew-Test-Org/Public-Test-Repo/runs/2124135546?check_suite_focus=true)